src/main/java/edu/kis/powp/jobs2d/command/CommandCounter.java

```diff
@@ -0,0 +1,23 @@
+package edu.kis.powp.jobs2d.command;
+
+public class CommandCounter implements CommandVisitor {
```

Comment:
Rename CommandCounter -> CommandCounterVisitor, because it is important to use this class according to Visitor pattern (execution by calling accept).